lib/navigation-bar/index.tsx

```diff
@@ -126,13 +126,12 @@ export class NavigationBar extends Component<Props> {
 
 const mapStateToProps: S.MapState<StateProps> = ({
   settings,
-  ui: { dialogs, openedTag, showNavigation, showTrash },
+  ui: { dialogs, showNavigation, collection },
 }) => ({
   autoHideMenuBar: settings.autoHideMenuBar,
   isDialogOpen: dialogs.length > 0,
-  openedTag,
+  collection,
```

Comment:
Ditto on the alphabetical order comment - and a bunch of other places too, which I am not going to be super annoying by leaving a line comment for each one 😂